main.py

```diff
@@ -1 +1,106 @@
+import pandas as pd
+import pyodbc
 
+from exceptions import DatabaseError
+
+class CsvToDatabase:
+
+    SERVER_NAME = 'Jiani_da\MSSQLSERVER04'
+    DATABASE_NAME = 'Python_SQL_CSV'
+    CREATE_TABLE_QUERY = f'''CREATE TABLE [dbo].[master_list] ( 
```

Comment:
the string "master_list" is used all over the class, we can make it a class Variable